### PR TITLE
create project in modal

### PR DESCRIPTION
**Pull request by vellip**

### CODE REVIEW

File: meinberlin/apps/contrib/static/js/ajax_modal.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/* globals $ */
+$(function () {
+  var $ajaxModals = $('[data-toggle="ajax-modal"]')
+  var modalHTML = (
+    '<div class="modal">' +
+      '<div class="modal-dialog modal-lg" role="document">' +
+        '<div class="modal-content">' +
+          '<div class="modal-header"><h2 class="modal-title u-first-heading"></h2><button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button></div>' +
```
Comment (xi) [inline meinberlin/apps/contrib/static/js/ajax_modal.js]: The "Close" label should be translated.

File: meinberlin/apps/contrib/static/js/ajax_modal.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          '<div class="modal-body"></div>' +
+      '  </div>' +
+      '</div>' +
+    '</div>'
+  )
+  var $backdrop = $('<div class="modal-backdrop" />')
+
+  $ajaxModals.on('click', function (e) {
```
Comment (xi) [inline meinberlin/apps/contrib/static/js/ajax_modal.js]: I would prefer to use event delegation here `$(document).on('click', '[data-toggle="ajax-modal"]', function() {})`

**Comment by vellip** [comment]
I noticed the thing with the validation errors too. But I think that's something we can live with. I'd not want to code JS Form validation.  That's out of scope.

**Comment by kleingeist** [comment]
i also think we should redirect to the blueprint list on cancel instead of the project list.

**Pull request merged**